src/pal/linux/mbuf.h

```diff
@@ -25,6 +25,7 @@ limitations under the License.
 #include <stdint.h>
 #include <string.h>
 #include <assert.h>
+#include <os_time.h>
```

Comment:
I think you want `#include <time.h>` here

Reply:
... and maybe `<types.h>` as well (but probably not)

Reply:
done